app/scripts/modules/core/src/serverGroup/configure/common/wizard/fields/ServerGroupDetailsField.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { FormikProps } from 'formik';
+
+import { Application, HelpField, IServerGroupCommand, Overridable } from 'core';
+
+export interface DetailsFieldProps<T extends IServerGroupCommand> {
+  app: Application;
```

Comment:
it doesn't look like `app` is being used, but I assume it is there for the override component to use?

Reply:
Correct, it is used there.